configs/platforms/sles-11-x86_64.rb

```diff
@@ -1,8 +1,8 @@
 platform "sles-11-x86_64" do |plat|
-  plat.servicedir "/etc/init.d"
+  plat.servicedir "/usr/lib/systemd/system"
   plat.defaultdir "/etc/sysconfig"
-  plat.servicetype "sysv"
-  plat.add_build_repository "http://osmirror.delivery.puppetlabs.net/sles-11-deps-x86_64/sles-11-deps-x86_64.repo"
```

Comment:
I think we want to keep this so that we install the packages below, like `gcc`, from our internal mirror.